container_engine/django_tutorial/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: all
+all: deploy
+
+.PHONY: create-cluster
+create-cluster:
+	gcloud container clusters create polls \
+		--scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform"
+
+.PHONY: create-bucket
+create-bucket:
+	gsutil mb gs://$(GCLOUD_PROJECT)
```

Comment:
Add the line to figure this out for them automatically: https://github.com/GoogleCloudPlatform/getting-started-python/pull/18/files#diff-854d0aab3e0cb4d4492bd8ecceecfb59R1